bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/LightThingHandler.java

```diff
@@ -316,12 +326,12 @@ public void messageReceived(String sensorID, DeconzBaseMessage message) {
         }
     }
 
-    private int unscaleColorTemperature(double ct) {
-        return (int) (ct / 100.0 * (500 - 153) + 153);
+    private int unscaleColorTemperature(final double ct) {
+        return (int) (ct / 100.0 * (ct_max - ct_min) + ct_min);
```

Comment:
```suggestion
        int ctMired =(int) (100000.0 / ct);
        if (ctMired > ctmax) {
            return ctmax;
        } else if (ctMired < ctmin) {
            return ctmin;
        } else {
            return ctMired;
        }
```

and the conversion in the other direction below. We can add teh dynamic state description later. You haver to adjust the documentation accordingly (color temperature is not 0->100 but xxx K -> xxx K. If you change that today, we can merge it before the 2.5.6 release and it is not a breaking change since 2.5.5 did not contain light support for deconz.